pkg/imageverifycache/client.go

```diff
@@ -75,19 +101,29 @@ func (c *cache) Set(ctx context.Context, policy kyvernov1.PolicyInterface, ruleN
 	if !c.isCacheEnabled {
 		return false, nil
 	}
-	c.logger.Info("Successfully set cache", "policy", policy.GetName(), "ruleName", ruleName, "imageRef", imageRef)
+	key := generateKey(policy, ruleName, imageRef)
+
+	stored := c.Cache.SetWithTTL(key, nil, 1, c.ttl)
+	if stored {
+		c.logger.Info("Successfully set cache", "policy", policy.GetName(), "ruleName", ruleName, "imageRef", imageRef)
+		return true, nil
+	}
 	return false, nil
 }
 
 func (c *cache) Get(ctx context.Context, policy kyvernov1.PolicyInterface, ruleName string, imageRef string) (bool, error) {
 	c.lock.Lock()
 	defer c.lock.Unlock()
-
 	c.logger.Info("Searching in cache", "policy", policy.GetName(), "ruleName", ruleName, "imageRef", imageRef)
 	if !c.isCacheEnabled {
 		return false, nil
 	}
+	key := generateKey(policy, ruleName, imageRef)
+	_, found := c.Cache.Get(key)
```

Comment:
you don't need the value ?

Reply:
Yeah, there is no use for the value. Ristretto Get() will return true only for the unexpired entry.

Reply:
Yeah, but the value could be `false` right ?

Reply:
If entry is expired or not present in cache then it will return false.